### PR TITLE
Defined Input Options for Loss and Optimizer

### PR DESCRIPTION
**Pull request by vasudev13**
Select options for defining loss and optimizer for network. However, we do need a learning rate input (probably a TextField accepting numbers).

The list of loss functions and optimzers is from official Keras Docs.

Thanks a lot

Peek 2020-03-26 17-30

### CODE REVIEW

**Review by orionpax00 — DISMISSED** [review]

**Comment by ivantha** [comment]
Closing as stale.
